docs/gettinginvolved/plugins/faq.rst

```diff
@@ -33,15 +33,14 @@ Is there a script to cleanup volumes leftover from running functional tests?
 -----------------------------------------------------------------------------
 
 Yes.
-After each test case, `detach_destroy_volumes <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/test/test_blockdevice.py>`_ is run automatically to cleanup volumes created by the test case.
-This cleanup call is added as part of `get_blockdeviceapi_with_cleanup <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/test/blockdevicefactory.py>`_ .
+After each test case, ``flocker.node.agents.testtools.detach_destroy_volumes`` will cleanup volumes created by the test case.
+This cleanup function is registered for you if you load your API using ``flocker.node.agents.testtools.get_blockdeviceapi_with_cleanup``.
 
-Please use ``get_blockdeviceapi_with_cleanup`` in your test wrapper.
 
 I get a lot of output in ``journactl`` and it’s very difficult to track what all is happening, is there an easy way to view the logs?
 -------------------------------------------------------------------------------------------------------------------------------------
 
-`eliottree` is the preferred way, but it currently does not work due to `a known bug <https://github.com/jonathanj/eliottree/issues/28>`_ . 
+`eliottree` is the preferred way, but it currently does not work due to `a known bug <https://github.com/jonathanj/eliottree/issues/28>`_ .
```

Comment:
That issue is resolved so you can actually just remove the "but it currently does not work" clause of this sentence.